README.md

```diff
@@ -63,3 +63,13 @@ The main differences between `dune-release` and `topkg` are:
 - Bundle everything as a single binary;
 - Use of `Astring`, `Logs`, `Fpath` and`Bos`;
 - Remove the IPC layer (which is used between `topkg` and `topkg-care`);
+
+### Runtime dependencies
+
+- `bzip2` can be set with the `DUNE_RELEASE_BZIP2` variable (deprecated), otherwise it expected to be in `PATH`;
+- `tar` can be set with the `DUNE_RELEASE_TAR` variable (deprecated), otherwise it is expected to be in `PATH`;
+- `git` can be set with the `DUNE_RELEASE_GIT` variable (deprecated), otherwise it is expected to be in `PATH`;
+- `hg` can be set with the `DUNE_RELEASE_HG` variable (deprecated), otherwise it is expected to be in `PATH`;
+- `curl` is expected to be in `PATH`.
```

Comment:
If `curl` is checked for by `dune-release`, maybe worth it to give `curly` an explicit binary, just to prevent unexpected behavior, maybe overkill though.

Reply:
That's right, I added this change to the PR.

Reply:
After being reviewed it may have been an overkill afterall, so I've reverted this change. Let's keep it this way as long as Curly works with "curl" as default value.